### DISCUSSION TITLE
Audio Transcript Mp3

### DISCUSSION THREAD

**Discussion by Jefresongil** [sepinf-inc/IPED#2018] [General]
Are there any restrictions for transcribing .mp3 audios? I'm processing a case with 10000 files, but to no avail. Includes in the Config file in mimeToprocess the audio/mp3 option. I must be commenting on some small mistake that is invisible to my eyes.

mimesToProcess = audio/3gpp; audio/3gpp2; audio/vnd.3gpp.iufp; audio/x-aac; audio/x-aiff; audio/amr; audio/amr-wb; audio/amr-wb+; audio/mp4; audio/mp3; audio/ogg; audio/vorbis; audio/x-oggflac; audio/x-oggpcm; audio/opus; audio/speex; audio/qcelp; audio/vnd.wave; audio/x-caf; audio/x-ms-wma; audio/x-opus+ogg; audio/ilbc

Testado com ambas
implementationClass = iped.engine.task.transcript.VoskTranscriptTask

implementationClass = iped.engine.task.transcript.Wav2Vec2TranscriptTask

Versão em uso 4.1.5

**Comment by lfcnassif** [comment]
> Are there any restrictions for transcribing .mp3 audios?

No, there isn't.

> Includes in the Config file in mimeToprocess the audio/mp3 option

The correct mime for MP3 audios is `audio/mpeg`

**Reply by Jefresongil** [reply]
Completely correct. And I had seen that when I looked in the metadata, but I didn't follow this intuitive. It's already processing. Thank you.

**Comment by gfd2020** [comment]
Wouldn't it be interesting to add the mp3 mime type (audio/mpeg) to the default configuration file?

**Reply by lfcnassif** [reply]
I think this is use case specific, the idea of not including it by default was to skip music. For example, mp3 files were a big issue for an user: https://github.com/sepinf-inc/IPED/issues/1230

**Reply by gfd2020** [reply]
I see. It makes sense. However, this file format is widely used (other than music). I helped a user with a specific case who was not transcribing mp3. I can add a note on the WIKI about this, what do you think?

**Reply by lfcnassif** [reply]
Sure, please!

**Reply by gfd2020** [reply]
I just updated the wiki. Then take a look, please. 
PS: Also see the CUDA tutorial...
